src/NServiceBus.Persistence.ServiceFabric/SagaPersister/RuntimeSagaInfo.cs

```diff
@@ -33,8 +33,8 @@ class RuntimeSagaInfo
 
             SagaAttribute = new ServiceFabricSagaAttribute
             {
-                CollectionName = !string.IsNullOrEmpty(userProvidedAttribute?.CollectionName) ? userProvidedAttribute.CollectionName : sagaDataType.Name,
-                SagaDataName = !string.IsNullOrEmpty(userProvidedAttribute?.SagaDataName) ? userProvidedAttribute.SagaDataName : sagaDataType.Name
+                CollectionName = userProvidedAttribute?.CollectionName ?? sagaDataType.Name,
```

Comment:
@tmasternak and I discussed this and concluded an empty string is not valid anyway as a collection name. Do we really need that validation?